src/workerskv/write_bulk.rs

```diff
@@ -0,0 +1,44 @@
+use crate::endpoint::{Endpoint, Method};
+
+/// Write Key-Value Pairs in Bulk
+/// Writes multiple key-value pairs to Workers KV at once.
+/// A 404 is returned if a write action is for a namespace ID the account doesn't have.
+/// https://api.cloudflare.com/#workers-kv-namespace-write-multiple-key-value-pairs
+pub struct BulkWrite<'a> {
+    pub account_identifier: &'a str,
+    pub namespace_identifier: &'a str,
+    pub params: BulkWriteParams<'a>,
+    pub bulk_key_value_pairs: Vec<KeyValuePair<'a>>,
+}
+
+impl<'a> Endpoint<(), BulkWriteParams<'a>, Vec<KeyValuePair<'a>>> for BulkWrite<'a> {
+    fn method(&self) -> Method {
+        Method::Put
+    }
+    fn path(&self) -> String {
+        // Need multiple cases for presence of ttl or expiration parameters.
+        format!("accounts/{}/storage/kv/namespaces/{}/bulk", self.account_identifier, self.namespace_identifier)
+    }
+    fn body(&self) -> Option<Vec<KeyValuePair<'a>>> {
+        Some(self.bulk_key_value_pairs.clone())
+    }
+    fn query(&self) -> Option<BulkWriteParams<'a>> {
+        Some(self.params.clone())
+    }
+    // default content-type is already application/json
+}
+
+#[derive(Serialize, Clone, Debug)]
+pub struct BulkWriteParams<'a> {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub expiration: Option<&'a str>,
```

Comment:
The way I understood the query params is that they were used as url params. I followed the use of Option because I wanted to make sure these params weren't included if their value was none. I was also trying to follow the example in dns.rs:
```
#[derive(Serialize, Clone, Debug, Default)]
pub struct ListDnsRecordsParams {
    #[serde(skip_serializing_if = "Option::is_none")]
    pub record_type: Option<DnsContent>,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub name: Option<String>,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub page: Option<u32>,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub per_page: Option<u32>,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub order: Option<ListDnsRecordsOrder>,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub direction: Option<OrderDirection>,
    #[serde(rename = "match", skip_serializing_if = "Option::is_none")]
    pub search_match: Option<SearchMatch>,
}
```
Let me know if I misunderstood how the query params are used--I think I did it the right way, but I am also new to this code base!

Reply:
Oh oops, I'm sorry, I meant to write "should these be Option<uint>", but forgot the uint part. We should make sure users can't put "asdf" for the expiration, is all. Not sure which uint would be appropriate, but the [DNS endpoints use u32](https://github.com/cloudflare/cloudflare-rs/blob/master/src/endpoints/dns.rs#L52)

Reply:
AFAIK Time since unix epoch is stored in i32, so we should probably be consistent! (Even though it really should be u32 imho)